devtools/jsonapi/main/main.go

```diff
@@ -1,87 +1,46 @@
-	"github.com/orbs-network/orbs-network-go/test/crypto/keys"
-	"github.com/orbs-network/orbs-spec/types/go/primitives"
-	"golang.org/x/crypto/ed25519"
+	"github.com/orbs-network/orbs-network-go/devtools/jsonapi/commands"
 	"os"
 )
 
-// orbs-json-client [--send-transaction | --call-method]=<json> --public-key=<pubkey> --private-key=<privkey> --api-endpoint=<http://....>
-func main() {
-	sendTransactionPtr := flag.String("send-transaction", "", "<json>")
-	callMethodPtr := flag.String("call-method", "", "<json>")
-	generateTestKeysPtr := flag.Bool("generate-test-keys", false, "generates a pair of TEST keys, public and private; NEVER use them in production")
-	verbosePtr := flag.Bool("v", false, "Show all related logs")
-
-	publicKeyPtr := flag.String("public-key", "", "public key in hex form")
-	privateKeyPtr := flag.String("private-key", "", "public key in hex form")
-
-	apiEndpointPtr := flag.String("api-endpoint", "http://localhost:8080", "<http://..../api>")
-
-	flag.Parse()
-
-	logger := log.GetLogger(log.String("api-endpoint", *apiEndpointPtr)).WithOutput(log.NewOutput(os.Stdout).WithFormatter(log.NewHumanReadableFormatter()))
-
-	if *sendTransactionPtr != "" {
-		if *verbosePtr {
-			logger.Info("sending transaction")
-		}
-
-		if *publicKeyPtr == "" || *privateKeyPtr == "" {
-			logger.Error("Public key or private key is incorrect")
-			return
-		}
-
-		decodedPublicKey, publicKeyDecodeError := hex.DecodeString(*publicKeyPtr)
-		decodedPrivateKey, privateKeyDecodeError := hex.DecodeString(*privateKeyPtr)
-
-		if publicKeyDecodeError != nil {
-			logger.Error("Could not decode public key from HEX", log.Error(publicKeyDecodeError))
-		}
+// gamma-cli start [-port=8080]
+// gamma-cli run call|send [-public-key=<pubkey>] [-private-key=<privkey>] [-host=<http://....>]
 
-		if privateKeyDecodeError != nil {
-			logger.Error("Could not decode private key from HEX", log.Error(privateKeyDecodeError))
-		}
-
-		keyPair := keys.NewEd25519KeyPair(primitives.Ed25519PublicKey(decodedPublicKey), primitives.Ed25519PrivateKey(decodedPrivateKey))
-
-		tx := &jsonapi.Transaction{}
-		if err := json.Unmarshal([]byte(*sendTransactionPtr), tx); err != nil {
-			logger.Error("could not unpack json", log.Error(err))
-		}
-
-		result, _ := jsonapi.SendTransaction(tx, keyPair, *apiEndpointPtr, *verbosePtr)
-
-		jsonBytes, _ := json.Marshal(result)
-		fmt.Println(string(jsonBytes))
-	} else if *callMethodPtr != "" {
-		if *verbosePtr {
-			logger.Info("calling method")
-		}
-
-		tx := &jsonapi.Transaction{}
-		if err := json.Unmarshal([]byte(*callMethodPtr), tx); err != nil {
-			logger.Error("could not unpack json", log.Error(err))
-		}
-
-		result, _ := jsonapi.CallMethod(tx, *apiEndpointPtr, *verbosePtr)
-
-		jsonBytes, _ := json.Marshal(result)
-		fmt.Println(string(jsonBytes))
-	} else if *generateTestKeysPtr {
-		publicKey, privateKey, _ := ed25519.GenerateKey(rand.Reader)
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("Welcome to gamma-cli")
```

Comment:
I think it would be preferable if we had separate README.md for gamma with all the examples. This way it would be easier to keep it up to date.